src/initializers/postgres.ts

```diff
@@ -0,0 +1,56 @@
+import requireInjected from '../require-injected';
+import { getLogger } from 'log4js';
+import type * as PgTypes from 'pg';
+
+const logger = getLogger('orka.postgres');
+let pool: PgTypes.Pool;
+
+export default function postgres(config) {
+  if (config.postgres?.url && !pool) {
+    const { Pool }: typeof PgTypes = requireInjected('pg');
+    pool = new Pool({
+      connectionString: config.postgres.url,
+      max: config.postgres.poolSize,
+      ssl: config.postgres.sslConfig
```

Comment:
🙃  We have faced issues with ssl many times. Can you double check that the implementation works both for dbs that require ssl: true (officedroid) and for localhost?